src/lib/pages/workflows-with-new-search.svelte

```diff
@@ -42,6 +42,8 @@
     $supportsAdvancedVisibility,
   );
 
+  export let columns: string[];
```

Comment:
@laurakwhit can we just define `columns` here? I don't think we need this to be a prop. The we can remove this change: https://github.com/temporalio/cloud-ui/pull/412/files#diff-7ccc1372d570c8ec2ce0c7fffb604f961c6c63ebe1ce97e4028f65533ac02fb7R6-R11

I'm changing how this will work in my new branch anyway. Since users will be able to configure the columns (and we'll persist that configuration), we can access the configuration via a store in any component where we need it.

Reply:
I had defined it higher up so we could pass it to the old search page to determine the `colspan`. But if we're not going to support the old search/table we can just hard code the `colspan` there and define it here.

I won't touch this here so I don't mess with your WIP branch, but feel free to change it there!